hydroshare/local_settings.template

```diff
@@ -0,0 +1,31 @@
+# Copy the contents of this file to a new file local_settings.py
+# Values here will serve as overrides when developing locally
+# local_settings.py will not be version controlled so storing keys is acceptable
+
+DEBUG = True
+
+# Local resource iRODS configuration
+USE_IRODS = True
+IRODS_ROOT = '/tmp'
+IRODS_ICOMMANDS_PATH = '/usr/bin'
+IRODS_HOST = 'hydrotest41.renci.org'
+IRODS_PORT = '1247'
+IRODS_DEFAULT_RESOURCE = 'hydrotest41Resc'
+IRODS_HOME_COLLECTION = '/hydrotest41Zone/home/hsproxy'
+IRODS_CWD = '/hydrotest41Zone/home/hsproxy'
+IRODS_ZONE = 'hydrotest41Zone'
+IRODS_USERNAME = 'hsproxy'
+IRODS_AUTH = 'proxywater1'
```

Comment:
didn't @phuongdm change this... and this isn't really a local iRODS configuration, it's for the one at hydrotest41.renci.org.  I vote we make it actually local by default.

Reply:
I didn't change it, I changed the password

Reply:
right, I was pointing out this password is wrong... And I believe you changed it because this one right here is checked into source control.

Reply:
These keys were buried in master/develop and repeated with changes to local_settings. Overall, we need to move out to os.getenv. This has already been socialized in Slack and on my radar.

Reply:
@hyi Now the template contains unique information, is your concern addressed? Also, based on conversations with Scott and me, are all of your comments addressed in this PR can you approve? Phuong and I are fixing the Jenkins build to adapt to not having a local_settings. It's taking longer, because at the same time he is addressing old Jenkins issues by building a new server.

Reply:
@sblack-usu Can you approve this PR or highlight where I am missing a requested change?